### PR TITLE
0.13: update docs, examples and some code

### PR DESCRIPTION
**Pull request by Orzelius**
Please make your way to the commits

### CODE REVIEW

File: docs/k8s-plugins/action-types/container.md [hunk 1]
```diff
@@ -4,40 +4,41 @@ order: 1
 ---
 
 {% hint style="info" %}
-The `container` module type is an abstraction that can be used by multiple plugins. [See here](../../other-plugins/container.md) for an in-depth guide on the module type itself. Continue reading for how to deploy it with the Kubernetes plugin.
+The `container` action type is an abstraction that can be used with multiple plugins. [See here](../../other-plugins/container.md) for an in-depth guide on the action type itself. Continue reading for more information on the container deploy action type that can be used with the Kubernetes plugin.
```
Comment (vvagaytsev) [inline docs/k8s-plugins/action-types/container.md:7]: A minor nit :) We use the `fixed width` style when we refer to the `container` build action type later in the doc. Would it be nicer to use it here as well?

File: docs/k8s-plugins/action-types/helm.md [hunk 2]
```diff
@@ -5,143 +5,156 @@ order: 3
...
+
+[`helm-pod` run](../../reference/action-types/Run/helm-pod.md)
+and  [`helm-pod` test](../../reference/action-types/Test/helm-pod.md) will create a fresh kubernetes workload and run your command in it.
+These actions are cached. This means that if garden will not rerun them if the version of the action hasn't changed. If a remote kubernetes
+cluster is used, test results are stored there which allows to share test results between the team or ci runs to decrease the number or re-runs.
+
+`helm-pod` don't have to depend on the deploy actions. The manifests are gathered from the rendered helm charts and deployed to the cluster.
 
-The same applies to any _tests_ that you specify. Take for example the `vote` module:
+Here's a test action from the [vote-helm example](https://github.com/garden-io/garden/tree/0.12.51/examples/vote-helm/vote/garden.yml.
```
Comment (vvagaytsev) [inline docs/k8s-plugins/action-types/helm.md:108]: This link points to the `0.12.51` tag of the `main` branch. That code revision has a module-based configuration. We can replace it with a relative link, it should also work fine with the GitBook.

File: docs/k8s-plugins/action-types/helm.md [hunk 3]
```diff
@@ -162,157 +175,79 @@ In this example, `some.key` is set to `"prod-value"` for the `prod` environment,
...
-```
-
-The base chart can also be any `helm` module (not just "base" charts specifically made for that purpose), so you have
-a lot of flexibility in how you organize your charts.
+**TODO: allow non-relative paths for the chart and then write this**
 
 ## Production environments
 
-You can define a remote environment as a `production` environment by setting the [production flag](../../reference/project-config.md#environmentsproduction) to `true`. This affects some default behavior when deploying `helm`  modules. See the [Deploying to production](../advanced/deploying-to-production.md) guide for details.
+You can define a remote environment as a `production` environment by setting the [production flag](../../reference/project-config.md#environmentsproduction) to `true`. This affects some default behavior when working with `helm`  actions. See the [Deploying to production](../advanced/deploying-to-production.md) guide for details.
```
Comment (vvagaytsev) [inline docs/k8s-plugins/action-types/helm.md:245]:
```suggestion
You can define a remote environment as a `production` environment by setting the [production flag](../../reference/project-config.md#environmentsproduction) to `true`. This affects some default behavior when working with `helm`  actions. See the [Deploying to production](../advanced/deploying-to-production.md) guide for details.
You can define a remote environment as a `production` environment by setting the [production flag](../../reference/project-config.md#environmentsproduction) to `true`. This affects some default behavior when working with `helm` actions. See the [Deploying to production](../advanced/deploying-to-production.md) guide for details.
```

Comment (vvagaytsev) [inline docs/k8s-plugins/action-types/helm.md:245]: Nit: extra space between `helm` and actions

File: docs/k8s-plugins/action-types/helm.md [hunk 4]
```diff
@@ -162,157 +175,79 @@ In this example, `some.key` is set to `"prod-value"` for the `prod` environment,
...
 
 ## Next steps
 
-Check out the full [helm module reference](../../reference/module-types/helm.md) for more details, and the [vote-helm](https://github.com/garden-io/garden/tree/0.12.51/examples/vote-helm) example project for a full project that showcases Garden's Helm support.
+Check out the full [action reference](../../reference/action-types/README.md) for more details 
+and the [vote-helm](https://github.com/garden-io/garden/tree/0.12.51/examples/vote-helm) example project for a full project
```
Comment (vvagaytsev) [inline docs/k8s-plugins/action-types/helm.md:250]: Same as in https://github.com/garden-io/garden/pull/4119/commits/a894ebf941599cac7498ded9bd5c281a3e9a4ce1#r1176082959

The link points to the example with the module-based config.

**Review by vvagaytsev — CHANGES_REQUESTED** [review]
Looks good, but some example link should be double-checked.
Also left a few non-blocking comments.

One more general note. We do not have any strong convention for spelling action kinds. Should we spell those like `Kind` (e.g. `Build`, `Deploy`, etc.)? Just to make a clear difference between the normal verbs and the action kinds we refer to. I'm not sure if this is necessary. In the most cases the meaning is clear enough from the context. It's more about having a unified spelling style.

**Comment by vvagaytsev** [comment]
@Orzelius I've removed the blocking changes request. Please check the links to the examples and update those to relative ones if necessary. Otherwise, it's good to go! 👍 

Thanks for a great job! 🚀 Let's get his merged.

File: docs/k8s-plugins/action-types/helm.md (hunk 3 above)
Comment (Orzelius) [reply docs/k8s-plugins/action-types/helm.md:245]: I'm not sure what you mean by fixed width tbh

File: docs/k8s-plugins/action-types/container.md (hunk 1 above)
Comment (Orzelius) [reply docs/k8s-plugins/action-types/container.md:7]: I'm not sure what you mean by fixed width tbh

**Pull request merged**